pycparser/c_parser.py

```diff
@@ -547,9 +547,17 @@ def p_external_declaration_4(self, p):
 
     def p_pp_directive(self, p):
         """ pp_directive  : PPHASH
+                          | PPPRAGMA
```

Comment:
I'd have a separate parsing rule for pragma now that it's supported, leaving the other one as it was